Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/ExecutionNotificationsBus.h

```diff
@@ -450,7 +507,7 @@ namespace ScriptCanvas
         void Visit(LoggableEventVisitor& visitor) override;
     };
 
-    using Breakpoint = TaggedParent<BreakpointTag, Signal>;
+    using Breakpoint = TaggedParent<BreakpointTag, AZ::EntityId>; // NodeId
```

Comment:
Only meaningful change there, a breakpoint is done on a node, we don't differentiate its input/outputs slots signals

Reply:
Seems odd that the breakpoint would be on the node rather than the entry point, a node can have different execution slots as input, will there be a way to know which one triggered the breakpoint?

Reply:
Yes we place a breakpoint on a node, but when said breakpoint is triggered we know from which slot it was triggered and can send it back to the editor in the message. We might want to support breakpoint only on slots in the future, but for now it seems overkill